src/blocks/wiki-embed/block.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * BLOCK: Call To Action.
+ */
+
+// Import block dependencies and components.
+import Edit from './edit';
+
+const { __ } = wp.i18n;
+const { registerBlockType } = wp.blocks;
+
+registerBlockType( 'ubc/wiki-embed', {
+	title: 'UBC Wiki Embed',
+	description: 'Gutenberg block to provide ability to embed page contents from any wiki resources',
```

Comment:
Sadly not _quite_ true, it depends on the list of sites that people set. However, let's for now say

"Adds ability to embed content from select UBC OpenWiki platforms such as wiki.ubc.ca" and see a live preview in the editor.